course/templates/course/_siblings.html

```diff
@@ -4,7 +4,11 @@
 <nav class="row siblings">
 
     {% if next %}
-    {% module_accessible next as accessible %}
+     {% if next.submittable %}
```

Comment:
The indentation is wrong here. Don't use a single space. Our indentation in HTML templates has been inconsistent, but you could follow the style already used in the same file. Here, you could keep `{% if next.submittable %}` on the same level with `{% if next %}` and indent `{% exercise_accessible next as accessible %}` with four additional spaces so that it shows it is inside the if.

Reply:
Identation is changed so that the number of spaces in the beginning of the row is multiple of 4.